src/js/cf-expandables.js

```diff
@@ -5,110 +5,255 @@
  * A public domain work of the Consumer Financial Protection Bureau
  */
 
-(function( $ ) {
+( function( $ ) {
 
-  $.fn.expandable = function( userSettings ) {
+  'use strict';
 
-    return $( this ).each(function() {
+  var _extend = $.extend;
 
-      var $this = $( this ),
-          $target = $this.find('.expandable_target').not( $this.find('.expandable .expandable_target') ),
-          $cueOpen = $this.find('.expandable_cue-open').not( $this.find('.expandable .expandable_cue-open') ),
-          $cueClose = $this.find('.expandable_cue-close').not( $this.find('.expandable .expandable_cue-close') ),
-          $content = $this.find('.expandable_content').not( $this.find('.expandable .expandable_content') ),
-          $groupParent = $this.parents('.expandable-group'),
-          accordion = $groupParent.length > 0 && $groupParent.data('accordion');
+  /**
+   * Returns function with throttled callback.
+   *
+   * @param {duration} integer - The title of the book.
+   * @param {callback} function - The title of the book.
+   * @returns {function} - Time delayed function
```

Comment:
fixed in https://github.com/sebworks/cf-expandables/commit/4f9e0c59ac1aa93385c859ca507afcd135f72d75